hidapi/hidapi.h

```diff
@@ -80,7 +80,7 @@ extern "C" {
 
 			/** Pointer to the next device */
 			struct hid_device_info *next;
-		};
+		} HID_DEVICE_INFO;
```

Comment:
this change is not related to what PR is really about - support usage pages on macOS
it also changes public API, and there is no real need for it, please undo

Reply:
Fixed